locations/spiders/burger_king.py

```diff
@@ -1,66 +1,173 @@
 # -*- coding: utf-8 -*-
-import csv
-
 import scrapy
 
-from locations.items import GeojsonPointItem
+from locations.dict_parser import DictParser
+from locations.geo import city_locations, point_locations
 
 
-class BurgerKingSpider(scrapy.Spider):
+class BurgerKing(scrapy.Spider):
```

Comment:
Let's keep the convention of suffixing the class name with "Spider" please:

```suggestion
class BurgerKingSpider(scrapy.Spider):
```

Reply:
Done.